deploy/basic/terraform/main.tf

```diff
@@ -62,7 +65,7 @@ locals  {
   
   instance_shape = "VM.Standard.E2.1.Micro"
 
-  availability_domain = 3
```

Comment:
What happen if the user has no limits on any AD? (e.g.: not on home region).

Reply:
deploy will fail

Reply:
can trials even switch regions ?

Reply:
yes. Another use case is that the user are on the home region, but is already using the Always-free resources

Reply:
When resources are already used, they will hit service limits. When they don’t have resources, ( like different region) they will get an error about the AD being invalid.